### ISSUE TITLE
any updates soon ?

### ISSUE THREAD

**Issue by raykai** [Crypta-Eve/provi-i#22]
Hi just wondering if there will be any updates to this program i like it but it seems to be stalled.

**Comment by Crypta-Eve** [comment]
There will be updates if needed to maintain current functionality. However my time is dedicated to the next major version of spyglass. I am unfortunately very busy with life atm so progress is slower than I would like.... But progress is happening....

**Comment by raykai** [comment]
any way we can try the next major version of spyglass... any betta for it ?

**Comment by Crypta-Eve** [comment]
yes and no.... Not one i would be happy for others to run at this stage
